redirect/index.php

```diff
@@ -2,5 +2,5 @@
 
 include __DIR__ . '/routes.php';
 $routeHandler = new Route();
-header('Location: ' . $routeHandler->getTargetOfSub(mb_strtolower($_GET['sub']))); // Function does check if GET parameter is set / empty
+header('Location: ' . $routeHandler->getTargetOfSub(mb_strtolower($_GET['sub']), mb_strtolower($_GET['subsub']))); // Function does check if GET parameter is set / empty
```

Comment:
Please move the `mb_strtolower` to the function. This really shouldn't be here in this class. (+TestCase? :+1: )